### DISCUSSION TITLE
[ERROR] in line 417: exit code 0: while executing command wget -q --show-progress $URL

### DISCUSSION THREAD

**Discussion by alcaan16** [tteck/Proxmox#3155] [Q&A] [answered]
hello people. It's my first time on proxmox and I can't install HA. I am using Proxmox VE Helper-Scripts and when I try to install it it pops up

[ERROR] in line 417: exit code 0: while executing command wget -q --show-progress $URL

help!!!

hardware firebat t8

![error](https://github.com/tteck/Proxmox/assets/165582966/781e9c42-a5b9-4345-a3a2-3f545c684df8)

**Comment by tteck** [comment]
🏁 The script runs without any problems from this end.

```
    __  __                        ___              _      __              __     ____  _____
   / / / /___  ____ ___  ___     /   |  __________(_)____/ /_____ _____  / /_   / __ \/ ___/
  / /_/ / __ \/ __ `__ \/ _ \   / /| | / ___/ ___/ / ___/ __/ __ `/ __ \/ __/  / / / /\__ \
 / __  / /_/ / / / / / /  __/  / ___ |(__  |__  ) (__  ) /_/ /_/ / / / / /_   / /_/ /___/ /
/_/ /_/\____/_/ /_/ /_/\___/  /_/  |_/____/____/_/____/\__/\__,_/_/ /_/\__/   \____//____/

Using Advanced Settings
Using HAOS Version: 12.3
Virtual Machine ID: 113
Using Machine Type: i440fx
Using Disk Cache: Write Through
Using Hostname: haos12.3
Using CPU Model: Host
Allocated Cores: 2
Allocated RAM: 4096
Using Bridge: vmbr0
Using MAC Address: 02:73:B9:C7:B7:13
Using Vlan: Default
Using Interface MTU Size: Default
Start VM when completed: no
Creating a HAOS VM using the above advanced settings
 ✓ Using thin-pool for Storage Location.
 ✓ Virtual Machine ID is 113.
 ✓ https://github.com/home-assistant/operating-system/releases/download/12.3/haos_ova-12.3.qcow2.xz
 ✓ Downloaded haos_ova-12.3.qcow2.xz
 ✓ Extracted KVM Disk Image
 ✓ Created HAOS VM (haos12.3)
 ✓ Completed Successfully!
```

**Reply by alcaan16** [reply]
any possible solution?

**Reply by tteck** [reply]
What's the outcome from visiting `https://github.com/home-assistant/operating-system/releases/download/12.3/haos_ova-12.3.qcow2.xz`?

**Reply by alcaan16** [reply]
> What's the outcome from visiting `https://github.com/home-assistant/operating-system/releases/download/12.3/haos_ova-12.3.qcow2.xz`?

it is ok

**Reply by alcaan16** [reply, marked as answer]
solved! Trying again and again it worked alone without changing anything...
thank you

**Reply by negadin** [reply]
Hey there, I'm not OP.

I am able to reach  https://github.com/home-assistant/operating-system/releases/download/12.4/haos_ova-12.4.qcow2.xz without trouble from my browser to download the image. 

Not sure how it self-resolved for them other than repeated attempts, but I wanted to let you know the same line caused a problem for me also.

_edit: figured it _out.__
This is my first go with proxmox, and I under provisioned the root LVM. I suspect I was running out of disk space on the install attempt. I reinstalled proxmox with a larger disk size for the initial installation and it ran fine.

```
Using Advanced Settings
Using HAOS Version: 12.4
Virtual Machine ID: 100
Using Machine Type: q35
Using Disk Cache: Write Through
Using Hostname: haos
Using CPU Model: Host
Allocated Cores: 2
Allocated RAM: 4096
Using Bridge: vmbr0
Using MAC Address: 02:B5:B0:05:3A:D4
Using Vlan: Default
Using Interface MTU Size: Default
Start VM when completed: yes
Creating a HAOS VM using the above advanced settings
 ✓ Using dirdisk_nvme1n1 for Storage Location.
 ✓ Virtual Machine ID is 100.
 ✓ https://github.com/home-assistant/operating-system/releases/download/12.4/haos_ova-12.4.qcow2.xz
haos_ova-12.4.qcow2.xz              59%[===================================>                          ] 202.75M  69.9MB/s    in 2.9s    

[ERROR] in line 417: exit code 0: while executing command wget -q --show-progress $URL
```
